### ISSUE TITLE
Wait DRDY bit got to "1"

### ISSUE THREAD

**Issue by apallo6** [STMicroelectronics/sths34pf80-pid#1]
Instead of waiting for the DRDY transition to "1," the code is doing the opposite:

```
    /* wait DRDY bit go to '1' */
      do {
        ret += sths34pf80_tmos_drdy_status_get(ctx, &status);

      } while (status.drdy != 0U);
```

loop condition should be:
`while (status.drdy == 0U);`

**Comment by avisconti** [comment]
> Instead of waiting for the DRDY transition to "1," the code is doing the opposite:
> 
> ```
>     /* wait DRDY bit go to '1' */
>       do {
>         ret += sths34pf80_tmos_drdy_status_get(ctx, &status);
> 
>       } while (status.drdy != 0U);
> ```
> 
> loop condition should be: `while (status.drdy == 0U);`

Yes, correct, sorry for the typo. I'll prepare a new driver version.

**Comment by avisconti** [comment]
Please let me know if this issue is fixed. If other problems still persists, I think we may open a different bug

**Comment by avisconti** [comment]
fixed by 7c88a3e
